### ISSUE TITLE
Highlight but not scrolling 

### ISSUE THREAD

**Issue by rahuldange09** [quire-io/scroll-to-index#23]
Highlight but not scrolling  to bottom items which are not visible.
I have
```dart
SingleChildScollView(
    child:  ListView.generate( // works with this 
         child : ListView.generate( /// but not this
          )
     )
) 
```

PS : I have added scrollcontroller to singlechildscrollview 
I read other issues but its not solving my problem

Even if I go to that bottom item and try it scrolls back to top area where it becomes invisible

**Comment by rahuldange09** [comment]
Well I fixed it by using Column instead of ListView

```dart
SingleChildScollView(
    child:  ListView.generate( // works with this 
         child : Column( /// but not this
          )
     )
) 
```

But  any workaround to use it with ListView?

**Comment by jerrywell** [comment]
you can just use `ListView` instead of `singleChildScrollView` + column. it's because the column view is not a on demand widget which means the performance is not good. using list view will let you easily separate different row for lazy init when enter viewport.

**Comment by rahuldange09** [comment]
> you can just use `ListView` instead of `singleChildScrollView` + column. it's because the column view is not a on demand widget which means the performance is not good. using list view will let you easily separate different row for lazy init when enter viewport.

Yes I really want to use listview only that why I'm asking solution for https://github.com/quire-io/scroll-to-index/issues/23#issue-564797710 this problem.
Please help.

**Comment by jerrywell** [comment]
Hi, i mean, why you don't use

```
ListView(
    children: ...
)
```

instead of the whole code block as following:

```
SingleChildScollView(
    child: ListView(
        children: ...
    )
) 
```

**Comment by rahuldange09** [comment]
Hi,
Still thats not solving my problem.
Problem occurs when I place listview inside listview.  but if i put column inside listview then it works

**Comment by jerrywell** [comment]
Got you point. so this is a nested CustomScrollView issue. 
merged to #22.
